apps/generics/actions.py

```diff
@@ -13,8 +19,7 @@ class Action:
     icon_class: str = None
     extra_context: dict = None
     required_permissions: list = dataclasses.field(default_factory=list)
-
-    display_condition: callable = None
+    display_condition: Callable[[HttpRequest, Context], bool] = display_condition
```

Comment:
Since these are generic actions, I would suggest leaving these as-is and moving the `display_condition` method above to [this annotations file](https://github.com/dimagi/open-chat-studio/blob/main/apps/annotations/tables.py) and then passing it as the `display_condition` param like this:
```python
actions.edit_action(url_name="annotations:tag_edit", display_condition=display_condition),
```
and 
```python
                actions.delete_action(
                    url_name="annotations:tag_delete",
                    confirm_message="Continuing with this action will remove this tag from any tagged entity",
                    display_condition=display_condition
                ),
```

Nit: Maybe also rename the `display_condition` to something more specific?

Reply:
[9b6c01f](https://github.com/dimagi/open-chat-studio/pull/440/commits/9b6c01f8dfe8bc7fb726f86447dd75f4ca53cb2b)